docs/reference/editor-settings.md

```diff
@@ -1,8 +1,79 @@
 # Editor settings
 
-The editor settings supported by ty's language server, as well as the settings specific to [ty's VS Code extension](https://github.com/astral-sh/ty-vscode/).
+The editor settings supported by ty's language server, as well as the settings specific to [ty's VS
+Code extension][ty-vscode].
 
-## `python.ty.disableLanguageServices`
+## Runtime settings
+
+These settings define the behavior of the language server while it is running. They can be changed
+dynamically without needing to restart the server.
```

Comment:
I'm not sure if we want to provide this kind of information to the user facing docs, happy to hear others thoughts.

Reply:
> I think I would remove _Runtime settings_ but keep _Initialization options_. I'm not a huge fan of the terminology because it isn't really clear for users what initialization options are. Given that the only supported options are logging related, should we rename the section to `Logging options` (and maybe move it to the very bottom?)

This is a good idea but I'm worried in the future we might have to rename it if there's anything other than logging related options that gets added. I think I'd keep it "initialization options" for now but I've updated the brief at the start to avoid using "language server" and change the language to be more towards using ty in an editor.